content/kots-cli/upload.md

```diff
@@ -5,4 +5,26 @@ title: kots upload
 weight: 90140
 ---
 
-If you have a copy of an application that was created with kots pull or kots download, you can upload it back to the Admin Console using the `kots upload` command.
+Upload Kubernetes manifests from the local filesystem to a cluster, creating a new version of the application that can be deployed. When you have a copy of an application that was created with `kots pull` or `kots download`, you can upload it back to the Admin Console using the `kots upload` command. 
+
+### Usage
+```bash
+kubectl kots upload [source] [flags]
```

Comment:
does this also require [namespace]

Reply:
namespace is included in the [flags]. Strictly speaking, not a required argument, so I left it out of the usage in this case.